docs/pages/machine-id/access-guides/ansible.mdx

```diff
@@ -31,7 +31,75 @@ You will need the following tools to use Teleport with Ansible.
 
 - (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/2. Configure Ansible
+## Step 1/4. Configure RBAC
+
+First, Teleport must be configured to allow the credentials produced by the bot
```

Comment:
I think it would make sense to add a short paragraph (or one sentence) at the top of the page (after the first paragraph) describing the architecture of the setup we achieve in this guide. This will clarify:

- What "the bot" means
- What "the SSH servers" mean and how the credentials produced by the bot connect to them via Teleport.

Reply:
Addressed in 4ec7753c9f